Hudl.Mjolnir/Breaker/FailurePercentageCircuitBreaker.cs

```diff
@@ -31,19 +32,21 @@ internal class FailurePercentageCircuitBreaker : ICircuitBreaker
 
         private readonly GroupKey _key;
         private readonly IStats _stats;
+        private readonly IMetricEvents _metricEvents;
 
         // ReSharper disable NotAccessedField.Local
         // Don't let these get garbage collected.
         private readonly GaugeTimer _timer;
+        private readonly GaugeTimer _timer2;
```

Comment:
Could we maybe rename this one to _metricsTimer, and the other one to _statsTimer? I was confused about why we had two, until I read the comment below.